### PR TITLE
Add support for image-maps

### PR DESCRIPTION
**Pull request by philipwhiuk**
For some reason it breaks - I'll fix it up.

### CODE REVIEW

File: k9mail/src/main/java/com/fsck/k9/message/html/HtmlSanitizer.java [hunk 1]
```diff
@@ -24,6 +24,10 @@
                 .addAttributes("td",
                         "align", "bgcolor", "colspan", "headers", "height", "nowrap", "rowspan", "scope", "valign",
                         "width")
+                .addAttributes("map", "name")
+                .addAttributes("area", "shape", "coords", "href", "alt")
```
Comment (cketti) [inline k9mail/src/main/java/com/fsck/k9/message/html/HtmlSanitizer.java]: Looks like there are a couple of other attributes we should add as well. See https://www.w3schools.com/tags/tag_area.asp

Comment (philipwhiuk) [reply k9mail/src/main/java/com/fsck/k9/message/html/HtmlSanitizer.java]: Yeah, I did look at I wasn't sure we actually wanted them all. Certainly I don't really want `target` for example. I'll have a look what gets allowed for `a` and add similar ones.

Comment (philipwhiuk) [reply k9mail/src/main/java/com/fsck/k9/message/html/HtmlSanitizer.java]: Apparently we only allow `"href", "title"` for `a`. So I think this is sufficient.

**Comment by cketti** [comment]
For future consideration: `domain.com` is an existing commercial domain. Please use something like `example.org` or `domain.example` for test data.

**Pull request merged**